banjo/conf/schema.xml

```diff
@@ -219,7 +219,7 @@
    <field name="speed" type="text_general" indexed="true" stored="true"/>
    <field name="thickness" type="text_general" indexed="true" stored="true"/>
    <field name="brand" type="text_general" indexed="true" stored="true"/>
-   <field name="duration" type="int" indexed="true" stored="true"/>
+   <field name="duration_seconds" type="long indexed="true" stored="true"/>

```

Comment:
As you are renaming this field I have two questions:
* Does Jelly currently set the duration field for indexing?
* Does banjo search this field?